soroban-sdk-auth/src/lib.rs

```diff
@@ -8,13 +8,48 @@ use crate::public_types::{
     SignaturePayloadV0,
 };
 
+/// Users of this module will need to pass a struct that implements NonceAuth
+/// NonceAuth should manage a nonce in the contracts storage
```

Comment:
Can we end sentences with a full stop for consistency with docs being fully formed sentences/paragraphs?
```suggestion
/// NonceAuth should manage a nonce in the contracts storage.
```

Reply:
done